proto/pbpeering/peering.proto

```diff
@@ -2,12 +2,11 @@ syntax = "proto3";
 
 package pbpeering;
```

Comment:
Did we want to rename this to remove the `pb` prefix from the gRPC wire protocol ?

Reply:
Reminder.

Reply:
Fixed